src/Rector/TypeAnalyzerTrait.php

```diff
@@ -40,6 +41,18 @@ public function autowireTypeAnalyzerDependencies(
     protected function isType(Node $node, string $type): bool
```

Comment:
So we are not renaming this method, to make it more clear what it does?

Reply:
This is just extension of `$type`input